data/registry/instrumentation-dotnet-nservicebus.yml

```
@@ -0,0 +1,24 @@
+# cSpell:ignore # add words unknown to cspell, remove this line if not required
+title: NServiceBus OpenTelemetry Integration # the name of your project
+registryType: instrumentation # the type of integration; is this an exporter, plugin, API package, or something else?
+language: dotnet
+tags:
+  - nservicebus
+  - dotnet
+urls:
+    website: https://particular.net/ # to work on the integrations page as well
+    repo: https://github.com/Particular/NServiceBus
+    docs: https://docs.particular.net/nservicebus/operations/opentelemetry
+license: Apache 2.0 # or whatever your OSS license is (see https://opensource.org/licenses/), proprietary or non-OSI approved may be rejected
+description: A friendly description of your integration/plugin
+authors:
+  - name: Particular Software
+  - url: https://github.com/Particular
```

```suggestion
  - name: Particular Software
     url: https://github.com/Particular
     email: contact@particular.net
```

see @cartermp's comment, this should fix the validation issues